src/models/session.ts

```diff
@@ -165,6 +169,10 @@ export class Session extends Model {
     return this._sessionData.endedAt !== null ? new Date(this._sessionData.endedAt) : null
   }
 
+  get sessionPlanSequenceInstance () {
+    return this._sessionData.sessionPlanSequenceInstance
+  }
+
```

Comment:
This implementation is not correct. It needs to be `eagerSessionPlanSequenceInstance` and it needs to instantiate  a `SessionPlanSequence` class.

Reply:
I added `SessionPlanSequenceInstance` class instead of `SessionPlanSequence`.  The instance class is different from the sequence.